src/domain/subscription/subscription.repository.ts

```diff
@@ -1,7 +1,13 @@
+import { ChannelInfo } from '../streamer/streamer.entity';
 import { SubscriptionWithChannelResult } from './result/subscription-with-channel.result';
+import { SubscriptionEntity } from './subscription.entity';
 
 export const SUBSCRIPTION_REPOSITORY = Symbol('SubscriptionRepository');
 
 export interface SubscriptionRepository {
-  getSubscriptions(): Promise<SubscriptionWithChannelResult[]>;
+  getSubscriptions(userId: number): Promise<SubscriptionWithChannelResult[]>;
+  findSubscription(userId: number, streamerId: number): Promise<SubscriptionEntity | null>;
```

Comment:
```suggestion
  getSubscriptions(userId: number): Promise<SubscriptionWithChannelResult[]>;
  getSubscription(userId: number, streamerId: number): Promise<SubscriptionEntity | null>;
```
위에서는 `getSubscriptions` 표현을 쓰셨는데, 밑에서는 `findSubscription` 표현을 쓰셔서 조금 어색한 것 같습니다.
`getSubscription` 을 쓰는게 변수명이 더 낫지 않을까요?

Reply:
조회 메서드는 `get` 접두사를 사용하도록 수정했습니다! 
감사합니다.🙂